File: Kattis/Planina.java
package planina;

import java.util.Scanner;

public class Planina {

    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int n, points, result;
        n = input.nextInt();
        
        points = (int) Math.pow(2, n) + 1;
        result = (int) Math.pow(points, 2); //((2^n + 1) ^2);
        System.out.println(result);
    }
    
}

File: Kattis/CSharp/Planina.cs
using System;

namespace Planina
{
    class Program
    {
        static void Main(string[] args)
        {
            int n, points;
            n = Convert.ToInt32(Console.ReadLine());
            points = (int) Math.Pow((Math.Pow(2, n) + 1), 2);
            Console.WriteLine(points);
            // for testing: Console.ReadKey();
        }
    }
}
